apps/condo/domains/resident/tasks/sendResidentMessage.task.js

```diff
@@ -0,0 +1,116 @@
+const { isArray, isEmpty, compact, get, omit, uniq } = require('lodash')
```

Comment:
File already putted into `tasks` folder. IMO its overhead to add `.task` prefix to it

Reply:
file could be moved somewhere sometime, so such suffix for tasks should stay anyway as I think